University of Leicester/Leicester210.xml

```diff
@@ -0,0 +1,326 @@
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+   schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="MfVK86307647" type="mss">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Vita of Qirqos, Vita of Gabra Krǝstos (= Alexis), Dǝrsān za-darasa qǝddus Yoḥannǝs ʾeṗṗis qoṗṗos za-ʾAksum baʾǝnta ʿǝbayu wa-gadlu la-qǝddus Yǝsḥāq, Prayer of the ninth hour</title>
+            <editor key="VP"/>
+            <editor key="AB" role="generalEditor"/>
+            <funder></funder>
+         </titleStmt>
+         <editionStmt><p>Cataloguing funded by the DFG Project
+               'The Gǝʿǝz Version of the Passio of St Cyricus (Gadla Qirqos): A Critical Edition, Translation, and Commentary', project number: 468455109; Principal investigator: Vitagrazia Pisani. The manuscript was identified during scoping carried out by the AHRC-DFG project Demarginalizing medieval Africa: Images, texts, and identity in early Solomonic Ethiopia (1270-1527), ref. no. AH/V002910/1.</p></editionStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+
+            <pubPlace>Hamburg</pubPlace>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                  licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+            </availability>
+
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+                  <msIdentifier>
+                     <repository ref="INS0981Leicester"/>
+                     <collection>Special collections</collection>
+                     <idno>Leicester, Library of the University College Leicester, 210</idno>
+                     <idno facs="https://specialcollections.le.ac.uk/digital/collection/p15407coll6/id/20000"></idno>
+                  </msIdentifier>
+                  <msContents>
+                    <summary></summary>
+   <msItem xml:id="ms_i1">
+     <locus from="2ra" to="26ra"/>
+     <title type="complete" ref="LIT1488GadlaQ">Gadla Qirqos</title>
+     <textLang mainLang="gez"/>
+     <incipit xml:lang="gez">
+     <locus target="#2ra"></locus>
+     በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ፡ ገድል፡ ወስምዕ፡ ዘቅዱስ፡ <persName ref="PRS3249Cyriacus">ቂርቆስ፡</persName>ቂርቆስ፡ ወዘእሙ፡ <persName ref="PRS5790Julitta">ኢየሉጣ፡</persName>ወዘካልአንኒ፡ ፼ወ፲፻ወ፬፻ወ፴፬ሰማዕትት፡ እለ፡ ተከለሉ፡ በመዋዕለ፡ እለ፡ <persName ref="PRS12019Eskenderos">እስክንድሮስ፡</persName> መኰንን፨ ወኮነ፡ ስደት፡ ዐቢይ፡ ለአብያተ፡ ክርስቲያናት፡ ዘክርስቶስ፨
+   </incipit>
+      <explicit type="ScribalNoteCompleting">Scribal note on completing of the work (following the explicit)
```

Comment:
I think the type here would be "expanded subscription" (https://betamasaheft.eu/Guidelines/?id=ColophSupplTit)

Reply:
I did in this way, as a colophon, and inside two different parts, what do you think?

<colophon xml:id="coloph1" xml:lang="gez">
    <locus from="26ra" to="26rb"></locus>
    <seg type="expanded">
    <locus from="26ra" to="26ra9"></locus>
<locus target="#26ra1"></locus>ተፈጸመ፡ ገድል፡ ወስምዕ፡ ዘቅዱስ፡ ቂርቆስ፡ ወዘእሙ፡ ኢየሉጣ፡ ወ፻፻ወ፬፻፴ወ፬በኂሩቱ፡ ለእግዚኣብሔር። ዘአጽንዖሙ፡ ለሰማዕት፡ ወአውረሶሙ፡ መንግሥተ፡ ሰማያት፨ አሜን፨
<note>The colophon starts with a scribal note on completing of the work.</note>
</seg>
<seg type="supplication">
<locus from="26ra10" to="26rb"></locus>
<locus target="#26ra10"></locus>ወከማሁ፡ ያብዝኅ፡ እግዚኣብሔር፡ ዐስቦ፡ በደብረ፡ ጽዮን፨ ለዘአጽሐፈ፡ ዘንተ፡ ገድለ፡ እኁነ፡ <persName ref="PRS13943BakweraSeyon" role="patron">በኵረ፡ ጽዮን</persName>፨ … <cb n="b4"/>ምስለ፡ ጸሐፊሁ፡ <persName ref="PRS13944NewayaMaryam" role="scribe">ንዋየ፡ ማርያም፡</persName> ነዳይ፡ ወምስኪን፨ ወጊጉይ፡ እምኵሎሙ፡ አባስያን፨ … ለዘጸሐፎ፡ ወለዘ፡ አጽሐፎ፡ ለዘአንበቦ፡ ወለዘተርጐሞ፡ ወለዘ፡ ሰምዐ፡ ቃላቲሁ፡ ኅቡረ፡ ይምሀሮሙ፡ እግዚአብሔር፡ በመንግሥተ፡ ሰማያት፡ አሜን፨
<note>The supplication mentions the commissioner and his name, በኵረ፡ ጽዮን፡ Bakʷǝra Ṣǝyon, and the scribe, ንዋየ፡ ማርያም፡ Nǝwāya Māryām. The final part of the note (end of <locus target="#26rb"></locus>) is in a compressed script. Few lines are also written over erasures.</note>
</seg>
</colophon>